release/Magarena/scripts/Disinformation_Campaign.txt

```diff
@@ -4,7 +4,7 @@ value=2.500
 rarity=U
 type=Enchantment
 cost={1}{U}{B}
-ability=When SN enters the battlefield, you draw a card and each opponent discards a card.;\
+ability=When SN enters the battlefield, you draw a card~each opponent discards a card.;\
```

Comment:
the `and` should be left in the text so that it reads properly, i.e. `draw a card~and each opponent...`

Reply:
fixed